### DISCUSSION TITLE
Question about ANC

### DISCUSSION THREAD

**Discussion by Nima0908** [pine64/OpenPineBuds#108] [Q&A]
Hello, i have a small question about the current ANC implementation. What kind of technice does it use? So does it use feedforward, feedback, or hybrid ANC techniques? Additionally, does it implement adaptive algorithms to enhance real-time noise cancellation performance? Iam very interested in trying to make the ANC better as ive heard its not that good currently.

**Comment by Ralim** [comment]
It is hybrid FF and FB.
No real time algo's as far as I know
